operators.py

```diff
@@ -123,20 +94,15 @@ def ensure_animated_prompts_text():
         text.select_set(0, 3, 0, -1)
 
 
-def ensure_animated_prompts_text_editor_in_workspace(context):
-    script_area = utils.get_area_by_type('TEXT_EDITOR', workspace_id=config.workspace_id)
+def ensure_animated_prompts_text_editor(context):
```

Comment:
I haven't actually used animated prompts, so either you can test that, or I can figure it out haha

Reply:
Okay found it quickly and this breaks, I'll fix it

Reply:
this is fixed now